README.md

```diff
@@ -0,0 +1,4 @@
+# glusterdocs
+
+Source code to gluster documentation: http://gluster.readthedocs.io
```

Comment:
@prashanthpai  I am mentioning this github repo is for  http://gluster.readthedocs.io. Usually there is a README saying about what this github repo is all about.